.github/workflows/ci-tests.yml

```diff
@@ -0,0 +1,48 @@
+# NOTE: This name appears in GitHub's Checks API and in workflow's status badge.
+name: ci-tests
+
+# Trigger the workflow when:
+on:
+  # A push occurs to one of the matched branches.
+  push:
+    branches:
+      - master
+  # Or when a pull request event occurs for a pull request against one of the
+  # matched branches.
+  pull_request:
+    branches:
+      - master
+
+jobs:
+  tests:
+    # NOTE: This name appears in GitHub's Checks API.
+    name: tests
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v2
+        with:
```

Comment:
I think it's safe to remove this work-around. It has been fixed for a while: https://github.com/actions/checkout/issues/237#issuecomment-667431246.

Reply:
But this is still not closed https://github.com/actions/checkout/issues/299?

Reply:
Yea, but it has been indicated that it is a duplicate of https://github.com/actions/checkout/issues/237 and I think it should be closed.